k8s/charts/openebs/README.md

```diff
@@ -61,19 +61,24 @@ helm del --purge <openebs-chart-name>
-| `apiserver.imageTag`                 | Docker Image Tag for API Server               | `0.5.4`                           |
-| `apiserver.replicas`                 | Number of API Server Replicas                 | `1`                               |
-| `provisioner.image`                  | Docker Image for Provisioner                  | `openebs/openebs-k8s-provisioner` |
-| `provisioner.imageTag`               | Docker Image Tag for Provisioner              | `0.5.4`                           |
-| `provisioner.replicas`               | Number of Provisioner Replicas                | `1`                               |
-| `jiva.image`                         | Docker Image for Jiva                         | `openebs/jiva`                    |
-| `jiva.imageTag`                      | Docker Image Tag for Jiva                     | `0.5.4`                           |
-| `jiva.replicas`                      | Number of Jiva Replicas                       | `3`                               |
+| Parameter                              | Description                                   | Default                           |
+| ---------------------------------------| --------------------------------------------- | --------------------------------- |
+| `rbac.create`                          | Enable RBAC Resources                         | `true`                            |
+| `image.pullPolicy`                     | Container pull policy                         | `IfNotPresent`                    |
+| `apiserver.image`                      | Docker Image for API Server                   | `openebs/m-apiserver`             |
+| `apiserver.imageTag`                   | Docker Image Tag for API Server               | `0.6.0-RC1`                       |
+| `apiserver.replicas`                   | Number of API Server Replicas                 | `1`                               |
+| `provisioner.image`                    | Docker Image for Provisioner                  | `openebs/openebs-k8s-provisioner` |
+| `provisioner.imageTag`                 | Docker Image Tag for Provisioner              | `0.6.0-RC1`                       |
+| `provisioner.replicas`                 | Number of Provisioner Replicas                | `1`                               |
+| `snapshotOperator.provisioner.image`   | Docker Image for Snapshot Provisioner         | `openebs/snapshot-provisioner`    |
+| `snapshotOperator.provisioner.imageTag`| Docker Image Tag for Snapshot Provisioner     | `0.6.0-RC1`                       |
+| `snapshotOperator.controller.image`    | Docker Image for Snapshot Controller          | `openebs/snapshot-provisioner`    |
+| `snapshotOperator.controller.imageTag` | Docker Image Tag for Snapshot Controller      | `0.6.0-RC1`                       |
+| `snapshotOperator.replicas`            | Number of Snapshot Operator Replicas          | `1`                               |
```

Comment:
RC2 ?

Reply:
Should we have to change Snapshot controller to Snapshot Operator?